### ISSUE TITLE
Regression in firefox tooltips

### ISSUE THREAD

**Issue by ammen99** [WayfireWM/wayfire#2139]
Reported on IRC:

```
<consolers> tooltips stop working in firefox after commit 78b560b3c965 "more complete input method support (#2053)"
<consolers> with that commit and fcitx, wezterm works but every keystroke calls xkbcomp
<consolers> where earlier you used to see "input method went away" on every keystroke
<consolers> i'm running with that patch reverted
```

**Comment by soreau** [comment]
Which version of Firefox? Tooltips seem to work fine here with wayfire 2d11be00 and Firefox 122.0.1 (64-bit).

**Comment by ammen99** [comment]
Do you have fcitx5 installed and configured @soreau? I haven't been able to repro but I assume fcitx5 must be relevant, otherwise the commit indicated by consolers shouldn't change anything indeed.

**Comment by soreau** [comment]
@ammen99 No, I haven't fcitx5. I guess it's good to be me 🐭

**Comment by ammen99** [comment]
consolers mentioned on irc that the bug has been fixed (probably by #2113)
